public/app/features/alerting/unified/components/rules/RulesGroup.tsx

```diff
@@ -75,6 +76,8 @@ export const RulesGroup = React.memo(({ group, namespace, expandAll, viewMode }:
     return isGrafanaRulerRule(rule.rulerRule) && rule.rulerRule.grafana_alert.provenance;
   });
 
+  const canEditGroup = hasRuler && !isProvisioned && !isFederated && canEditRules(rulesSourceName);
```

Comment:
can we edit groups that contain rules provided by a plugin?

Reply:
Good point! No, we cannot - technically we can, but we shouldn't allow that in the UI